llvm/test/CodeGen/Mips/cheri/cheri-global-bounds-hybrid-addrspacecast.ll

```diff
@@ -5,6 +5,7 @@
 
 ; ModuleID = 'global.c'
 
+; Note: under-aligned -> will need two csw instructions
```

Comment:
I think we want to make this `align 8` like the corresponding purecap test, and instead add new tests for unaligned accesses in both.

Reply:
Yes will do before merging.